commands/commands.js

```diff
@@ -3,31 +3,107 @@ import promptSync from "prompt-sync";
+import { dirname } from 'path';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+
 
 export const prompt = promptSync();
 
-const showWelcomeBanner = () => {
-    console.log("\n \n " + chalk.green(constantValues.title) + "\n \n" + chalk.bgGreen(constantValues.description) + "\n\n" + constantValues.jokeCategories + chalk.red(shortcuts.see_joke_categories) + "\n" + constantValues.backToStart + chalk.red(shortcuts.go_back_to_start) + "\n" + constantValues.exitProgram + chalk.red(shortcuts.exit_project) + "\n" + constantValues.viewJokesDatabase + chalk.red(shortcuts.view_jokes_database) + "\n");
+export const showAppTitle = () => {
+    console.log("\n \n " + chalk.green(constantValues.title));
+}
+
+export const showWelcomeBanner = () => {
+    console.log("\n \n" + chalk.bgGreen(constantValues.description) + "\n\n" + constantValues.jokeCategories + chalk.red(shortcuts.see_joke_categories) + "\n" + constantValues.exitProgram + chalk.red(shortcuts.exit_project) + "\n" + constantValues.viewJokesDatabase + chalk.red(shortcuts.view_jokes_database) + "\n" + constantValues.askForHelp + chalk.red(shortcuts.ask_for_help) + "\n");
 }
 
 export const askName = () => {
-    showWelcomeBanner();
     console.log(chalk.green("What is your name?"));
     const name = prompt(">>");
+    finalData.name = name;
     return name;
 }
 
+const storeData = () => {
+    const userDetails = JSON.stringify(finalData);
+    const readerFile = fs.existsSync(`${__dirname}/../files/${finalData.name}.json`, "utf-8");
+
+    if (!readerFile) {
+        // console.log("doesntExist");
+        // console.log(readerFile);
+        fs.writeFileSync(`${__dirname}/../files/${finalData.name}.json`, userDetails, (err) => {
+            if (err) {
+                console.log("error")
+            }
+            console.log("\n" + "Your jokes are saved on " + chalk.blue(finalData.name) + chalk.blue(".json") + ". Please enter " + chalk.blue("view") + " to view the file.");
+
+        });
+        console.log("\n" + "Your jokes are saved on " + chalk.blue(finalData.name) + chalk.blue(".json") + ". Please enter " + chalk.blue("view") + " to view the file.");
+    }
+    else {
+        // console.log("exists");
+        // console.log(userDetails);
+        fs.appendFileSync(`${__dirname}/../files/${finalData.name}.json`, userDetails, (err) => {
+            if (err) {
+                console.log("error")
+            }
+        })
+        console.log("\n" + "Your jokes are saved on " + chalk.blue(finalData.name) + chalk.blue(".json") + ". Please enter " + chalk.blue("view") + " to view the file.");
+    }
+
+
+
+}
+const readStoredData = () => {
+    const storedData = fs.readFileSync(`${__dirname}/../files/${finalData.name}.json`, "utf-8");
+    const stringifiedStoredData = `${storedData}`;
+    const parsedStoredData = JSON.parse(stringifiedStoredData);
+    // console.log(parsedStoredData);
+
+    console.table([parsedStoredData]);
```

Comment:
Maybe we could do something like this . 
```suggestion
    const storedDataArray = [];
    const storedData = fs.readFileSync(`${__dirname}/../files/${finalData.name}.json`, "utf-8");
    const stringifiedStoredData = `${storedData}`;
    stringifiedStoredData.split("\n").forEach((sd) => {
        console.log(sd)
        storedDataArray.push(JSON.parse(sd))
    })
    console.table(storedDataArray);
```